### PR TITLE
Add Calcy IV Android app to Others tools

### PR DESCRIPTION
**Pull request by Letsan**
Added this app that displays a button over your screen so you can check your mons' stats and level in one tap (Android).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -132,6 +131,7 @@ Please check the <a href="https://github.com/tobiasbueschel/awesome-pokemon/blob
 - [Server Status](http://www.mmoserverstatus.com/pokemon_go) - Shows you if Pokémon GO is down.
 - [Egg Statistics](https://app.cmmcd.com/) - Pokémon GO Egg Reporter.
 - [Poke Assistant](https://pokeassistant.com/) - Collection of calculators for Pokémon GO.
+- [Calcy IV](https://play.google.com/store/apps/details?id=tesmath.calcy) - One-tap stats and level calculation on Pokemon GO (Android).
```
Comment (tobiasbueschel) [inline readme.md]: Thanks for the PR @Letsan 🙏 Would you mind to correct the spelling of Pokémon here? Then I'll merge it in ASAP.

Sorry for the nitpick!

Comment (Letsan) [reply readme.md]: I just fixed it ;)

**Comment by Letsan** [comment]
Done! Sorry about that, a VS Code plugin for Markdown was interfering with the table of contents.

**Comment by tobiasbueschel** [comment]
> Done! Sorry about that, a VS Code plugin for Markdown was interfering with the table of contents.

No worries, thanks a lot! I've merged it now 🎉

**Pull request merged**